.github/workflows/check-by-name.yml

```diff
@@ -18,13 +18,6 @@ on:
 
 permissions: {}
 
-# Create a check-by-name concurrency group based on the pull request number. if
-# an event triggers a run on the same PR while a previous run is still in
-# progress, the previous run will be canceled and the new one will start.
```

Comment:
Can you replace this with a comment saying why we don't want a concurrency group? Otherwise somebody might just want to add this again in the future :)

Ideally also link to the github feature requests